build/Microsoft.DotNet.Cli.BranchInfo.props

```diff
@@ -1,6 +1,7 @@
 <Project>
   <PropertyGroup>
     <Channel>master</Channel>
+    <CoreSetupChannel>master</CoreSetupChannel>
```

Comment:
Ok, I'll create a new CoreSetup props file. For now it'll just have this one property, but in later PRs as we get to the other files we can move more properties there.